package.json

```diff
@@ -2,7 +2,7 @@
   "name": "hydrogen-starter",
   "private": true,
   "sideEffects": false,
-  "version": "1.5.6",
+  "version": "1.5.7",
```

Comment:
Only nit pick here is that I would change to a minor bump. 1.6.0 becuase these change maayyy introduce a breaking change to people